pkl-core/src/main/java/org/pkl/core/packages/PackageResolvers.java

```diff
@@ -490,7 +490,8 @@ private Path getMetadataPath(
       if (Files.exists(cachePath)) {
         return cachePath;
       }
-      var tmpPath = tmpDir.resolve(metadataRelativePath);
+      Files.createDirectories(tmpDir);
```

Comment:
Why wasn't this necessary before, even though `tmpDir` hasn't changed?

Reply:
`Files.createTempFile` throws if the temp dir doesn't exist.